src/vs/workbench/parts/debug/electron-browser/debugEditorContribution.ts

```diff
@@ -183,19 +193,104 @@ export class DebugEditorContribution implements IDebugEditorContribution {
 	}
 
 	private onFocusStackFrame(stackFrame: IStackFrame): void {
-		if (!stackFrame || stackFrame.source.uri.toString() !== this.editor.getModel().uri.toString()) {
+		// TODO: onFocusStackFrame isn't called for the first time when editor url changes due to source mapping
+		//console.log("onFocusStackFrame", this.editor.getModel().uri.toString(), this.editor.getModel().uri.toString(), stackFrame);
+
+		if (!stackFrame || stackFrame && stackFrame.source.uri.toString() !== this.editor.getModel().uri.toString()) {
```

Comment:
It seems you commented on an older review. Does not apply to current the iteration .